contracts/CO30_Rob_Byelomorye.IslaPera/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,4 @@
+The Fuerzas Armadas Revolucionarias Socialistas has attacked the Columbian state of Isla del pájaro azul, took a number of hostages,
+as well as placed explosives on the island's hydroelectric dam. The group is threatening to execute the hostages
+and blow the dam if the state does not immediately release the groups imprisoned leaders by 0830hrs. Synixe has been hired to
+resolve this matter.
```

Comment:
```suggestion
The Fuerzas Armadas Revolucionarias Socialistas has attacked the Columbian state of Isla del Pájaro Azul, and took a number of hostages, as well as placed explosives on the island's hydroelectric dam. The group is threatening to execute the hostages and blow the dam if the state does not immediately release the groups imprisoned leaders by 0830hrs. Synixe has been hired to resolve this matter.
```

You don't need to move things to a new line. The editor you are using should have an option for word wrap.